<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>328</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolTip {
    background-color: #2e2e2e;
    color: #ffffff;
    border: 1px solid #5b5e60;
    padding: 4px;
    border-radius: 4px;
    opacity: 200;
    font-size: 12px;
}

QLabel {
    color: #ffffff;
}

QFrame {
    background-color: #2e2e2e;
}

QPushButton {
    background-color: #3c3f41;
    color: #ffffff;
    border: 1px solid #5b5e60;
    border-radius: 4px;
	padding: 0px 0px;
}

QPushButton:hover {
    border-color: #4a90e2;
}

QPushButton:pressed {
    background-color: #3d78b2;
    border-color: #2a5f92;
}

QRadioButton {
    color: #ffffff;
    spacing: 5px;
}

QRadioButton::indicator {
    width: 12px;
    height: 12px;
    background-color: #2e2e2e;
    border: 1px solid #5b5e60;
    border-radius: 4px;
}

QRadioButton::indicator:hover {
    background-color: #4a90e2;
    border-color: #3d78b2;
}

QRadioButton::indicator:checked {
    background-color: #4a90e2;
    border-color: #3d78b2;
}

QRadioButton::indicator:pressed {
    background-color: #3d78b2;
    border-color: #2a5f92;
}

QRadioButton::indicator:checked:pressed {
    background-color: #3d78b2;
    border-color: #2a5f92;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QLabel" name="title_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: #3c3f41;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	padding-left: 5px;
}</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/ico/Icons/ico/phils-hub.ico</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="msg_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16</height>
            </size>
           </property>
           <property name="text">
            <string>Options Dialog...</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignmentFlag::AlignVCenter">
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">QFrame {
	background: transparent;
	border-bottom-left-radius: 9px;
	border-bottom-right-radius: 9px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="option1">
              <property name="text">
               <string>RadioButton</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="option2">
              <property name="text">
               <string>RadioButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="option3">
              <property name="text">
               <string>RadioButton</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="submit_btn">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-top-left-radius: 0px;
	border-top-right-radius: 0px;
}</string>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
